### PR TITLE
small cleanup

### PR DESCRIPTION
**Pull request by Maarten-vd-Sande**
I made some small cleanup changes. Mainly involves removing unused code (at least I think it wasn't used). One small other change I made was is that I changed the samples.tsv to contain public samples. This is necessary for the docs to make a DAG image. 

Once the tests are back to normal we can merge everything and should have a working scRNA workflow! Then still the documentation and the notebook(s) ofcourse :)

### CODE REVIEW

**Review by siebrenf — COMMENTED** [review]
Looks good as far as I can see. The removed code in get_genome was indeed unused.

We should add Tilman and Rebecca to the .zenodo.json!

File: docs/conf.py [hunk 1]
```diff
@@ -18,7 +18,8 @@
 
 project = "seq2science"
 author = "Maarten van der Sande & Siebren Frölich"
-copyright = "Maarten van der Sande, Siebren Frölich, Jos Smits, & Simon van Heeringen."
+copyright = "Maarten van der Sande, Siebren Frölich, Jos Smits, Rebecca Snabel, Tilman Schaeffers, & " \
```
Comment (tilschaef) [inline docs/conf.py:21]: Schaefers :)

File: seq2science/rules/configuration_explain.smk [hunk 2]
```diff
@@ -99,7 +99,7 @@ else:
         "decoy_transcripts": "Decoy transcript were generated in order improve improve Salmon indexing accuracy (using the script from https://github.com/COMBINE-lab/SalmonTools)",
         "salmon_quant": "Transcript abundances were quantified with Salmon v@salmon[salmon] (https://doi.org/10.1038/nmeth.4197) with options '{config[quantifier_flags]}'.",
         "htseq_count": "Read counting and summarizing to gene-level was performed on filtered bam using HTSeq-count v@gene_counts[htseq] (https://doi.org/10.1093/bioinformatics/btu638).",
-        "kallistobus-count": "TODO: what is happening in kallistobus?!",
+        "kallistobus-count": "Reads were aligned and transformed to bus format with kb-python v@kallistobus[kb-python], a tool that wraps kallisto (https://doi.org/10.1038/nbt.3519) and bustools (doi.org/10.1101/673285).",
```
Comment (tilschaef) [inline seq2science/rules/configuration_explain.smk:102]: a python wrapper for kallisto and bustools?

**Review by tilschaef — COMMENTED** [review]
- Grammar surname (Schaeffers -> Schaefers)
- Edit configuration_explain

**Review by tilschaef — APPROVED** [review]

**Pull request merged**